index.html

```diff
@@ -2343,6 +2343,13 @@ <h1>Representations</h1>
 behavior for any such properties defined by the representation.
     </p>
 
+<p class="note" title="Representation properties lexical and value space">
+It is recommended that representations SHOULD use the lexical representation 
+of a registered data types (e.g. JSON-LD uses XML Schema date), 
+but representation MAY also choose to represent the data types 
+differently (e.g. CBOR may represent the same property with a tag).
```

Comment:
```suggestion
It is RECOMMENDED that representations use the lexical representation 
of registered data types. For example, JSON and JSON-LD use the XML 
Schema dateTime lexical representation to represent <a>datetimes</a>. 
A representation MAY choose to represent the data types differently.
For example, CBOR-LD represents some <a>datetime</a> values 
using integers since the Unix epoch.
```